src/smexperiments/api_types.py

```diff
@@ -369,12 +448,6 @@ class TrialComponentSearchResult(_base_types.ApiObject):
     display_name = None
     source = None
     status = None
-    start_time = None
-    end_time = None
-    creation_time = None
-    created_by = None
-    last_modified_time = None
-    last_modified_by = None
```

Comment:
these are in the response see [DescribeExperiment.xml](https://code.amazon.com/packages/IronmanApiServiceModel/blobs/88f72c626422dfd2b4ee994c397c826baf7d692b/--/model/eureka/operations/DescribeExperiment.xml#L34-L37)

Reply:
yeah, I removed those since those will be covered by **kwargs